tests/scripts/test_lochness_create_template.py

```diff
@@ -33,9 +33,19 @@ def test_create_lochness_template(args):
 
 
 
-
 def test_create_lochness_template_multiple_study(args):
     args.studies = ['StudyA', 'StudyB']
     create_lochness_template(args)
     pass
     # create_lochness_template(args)
+
+
+def test_create_lochness_template_for_documentation(args):
```

Comment:
So do we need some templates to include in the documentation? If yes, it shouldn't be part of the testing infrastructure! But if including that part here has made our lives easy, feel free to ignore my comment.

Reply:
this is the testing function for the `script/lochness_create_template.py`